admin/class-boldgrid-backup-admin-upload.php

```diff
@@ -141,6 +141,17 @@ public function verify_upload_access() {
 			return false;
 		}
 
+		if ( ! preg_match( '/boldgrid-backup-.*-\d{8}-\d{6}/', $_FILES['file']['name'] ) ) {
```

Comment:
When you download a zip using the transfer method, the filename gets "uploaded" added. Please see https://github.com/BoldGrid/boldgrid-backup/blob/df87accc1538fdd28f6a3691996f688c1ec49d35/admin/class-boldgrid-backup-admin-upload.php#L252-L253

We also have another method that may be able to be used instead: https://github.com/BoldGrid/boldgrid-backup/blob/master/admin/class-boldgrid-backup-admin-archive.php#L377

Reply:
One example of what you were saying would be this filename ( boldgrid-backup-f6b7c7c2-uploaded-rcfix.imh-ip.com-20200415-165358.zip ) . In which case it still matches the regex

Reply:
That method will not work if the file being uploaded was from a different Boldgrid Backup site, since the identifier will be different